docs/cr_HarborCluster_spec.md

```diff
@@ -0,0 +1,203 @@
+## Harbor-Cluster CRD
+
+Below is the spec defined for the CR `harbor-cluster`:
+
+```yaml
+# harbor version to be deployed
+# this version determines the image tags of harbor service components
+# required
+version: 1.10
+
+# external URL for access Harbor registry
+# required
+publicURL: https://harbor.registry.com
+
+# password for the root admin 
+# required
+adminPasswordSecret: adminSecret
+
+# secret reference for the TLS certs
+tlsSecret: tlsSecret
+
+# certificate issuers
+certificateIssuerRef: cert_issuer
+
+paused: false
+
+priority: 30
+
+# pod instance number
+# required
+replicas: 3
+
+# source registry of images
+imageSource:
+  registry: harbor.com
```

Comment:
Is `registry` field is redundant? `imagePullSecrets` already contain the registry address.

Reply:
I don't think so. This registry field is used for defining the image pulling path. Even the secret contains the related registry path, I don't think it's suitable to refer to the code.

Reply:
core operator:

```yaml
  components:
    harbor-core:
      image: <your_repo>/repo:tag
```